packages/devextreme/testing/tests/DevExpress.ui.widgets/resizeHandle.tests.js

```diff
@@ -104,6 +104,26 @@ QUnit.module('Behavior', moduleConfig, () => {
         });
     });
 
+    ['prev', 'next', 'both', 'none'].forEach((scenario) => {
+        QUnit.test(`double click handler (${scenario} button visible)`, function(assert) {
+            const onCollapsePrevStub = sinon.stub();
+            const onCollapseNextStub = sinon.stub();
+            const onCollapsePrevCallCount = scenario === 'prev' || scenario === 'both' ? 1 : 0;
+            const onCollapseNextCallCount = scenario === 'next' ? 1 : 0;
+
+            this.reinit({
+                onCollapsePrev: onCollapsePrevStub,
+                onCollapseNext: onCollapseNextStub,
+                showCollapsePrev: scenario === 'prev' || scenario === 'both',
```

Comment:
Currently, there are no tests covering the runtime changes to the showCollapsePrev and showCollapseNext options